ecommerce-redpanda/compose.yaml

```diff
@@ -57,27 +52,27 @@ services:
       redpanda: {condition: service_healthy}
       mysql: {condition: service_healthy}
   debezium_deploy:
-    image: debezium/connect:1.8
+    image: debezium/connect:1.9
     depends_on:
       debezium: {condition: service_healthy}
     environment:
-      KAFKA_ADDR: redpanda:9092
+      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
+      - KAFKA_ADDR=${EXTERNAL_IP:-redpanda}:9092
     volumes:
-      - ../ecommerce/mysql/mysql_dbz.sh:/mysql_dbz.sh
+      - ./mysql/mysql_dbz.sh:/mysql_dbz.sh
     entrypoint: [bash, -c, /mysql_dbz.sh]
-  metabase:
-    image: metabase/metabase:v0.41.5
-    depends_on: [materialized]
-    ports:
-      - 3030:3000
+  # metabase:
```

Comment:
Since we are commenting out Metabase from the docker compose. Should we update the README and the diagram to remove it? Or change the language in how we refer to using Metabase

Reply:
Ah my bad, just re-enabled metabase